kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -415,9 +415,9 @@ def _insert_unique_keys(
         columns added to each DataFrame
         """
         self.logger.info('Begin unique key creation for standard concepts ...')
-        for extract_config_url, (source_file_url, df) in df_dict.items():
+        for reference, (df_name, df) in df_dict.items():
```

Comment:
I understand why you're renaming extract_config_url to reference (bc of #357) , but what is df_name? Isn't this still source_file_url?

Reply:
> Isn't this still source_file_url?

Nope. "Transform Module Output: default df"

Reply:
Oh I see, this is for the intermediate data so there is no source file url 👍

Reply:
![Screen Shot 2019-07-19 at 10 43 48 AM](https://user-images.githubusercontent.com/201996/61543795-28135680-aa12-11e9-923e-c59cd8ef9a07.png)